main.cpp

```diff
@@ -73,7 +83,28 @@ int main(int argc, char **argv)
         std::cerr << "Single point of failure disabled\n";
     }
 
-    run(configPath);
+    for (int i = 1; i < argc; i++)
+    {
+        if (std::strcmp(argv[i], "--file") == 0 || std::strcmp(argv[i], "-f") == 0)
+        {
+            configPath = argv[i + 1];
+            i++;
+        }
+        else if (std::strcmp(argv[i], "--loose") == 0 || std::strcmp(argv[i], "-l") == 0)
+        {
+            spofEnabled = false;
+        }
+        else
```

Comment:
Please also add --debug option, to set debugEnabled to true.

Reply:
Done.